scripts/upgrade_admin_server.sh

```diff
@@ -100,6 +100,20 @@ n.save"
 
     # Upgrade the distribution non-interactively
     zypper --no-color --releasever 12.3 ref -f
+
+    systemctl stop rabbitmq-server
+    systemctl stop couchdb
+    sleep 5
```

Comment:
why the sleep? I thought systemctl stop would hang until it stops or fails (timeout/failure to stop)

Reply:
Added it just for testing.

Reply:
Yeah, this might not be necessary